app/PharBuilder.php

```diff
@@ -353,7 +353,7 @@ public function buildPhar()
 
         $this->stubFile = $this->makePathRelative($this->stubFile);
         $this->phar->setStub(
-            (!$this->isSkipShebang() ? '#!/usr/bin/env php' . PHP_EOL : '') .
+            (!$this->isSkipShebang() ? '#!/usr/bin/env php' . "\n" : '') .
```

Comment:
Good catch, I forgot that Windows set `PHP_EOL` to `\r\n` and this line is Unix exclusive